src/dist/conf/config.yaml

```diff
@@ -28,3 +28,10 @@ omero.web:
 # Redis instance that OMERO.web is using to store session information
 redis:
     uri: "redis://password@localhost:6379/1"
+# Information about the session store.
+session-store:
+    #type is either "postgres" or "redis"
+    type: "postgres"
+    #synchronicity is either "sync" or "async"
+    synchronicity: "async"
+    db.url: "jdbc:postgresql://localhost:5432/omero_database?user=omero&password=omero"
```

Comment:
Need to update this to `uri` and add documentation of the possible URI parameters. Maybe add a Redis example using the URI above?